WinRT.Runtime/Projections/Bindable.net5.cs

```diff
@@ -0,0 +1,1395 @@
+        IBindableVectorView GetView();
+        bool IndexOf(object value, out uint index);
+        void SetAt(uint index, object value);
+        void InsertAt(uint index, object value);
+        void RemoveAt(uint index);
+        void Append(object value);
+        void RemoveAtEnd();
+        void Clear();
+        uint Size { get; }
+    }
+    [global::WinRT.WindowsRuntimeType]
+    [Guid("346DD6E7-976E-4BC3-815D-ECE243BC0F33")]
+    internal interface IBindableVectorView : IEnumerable
+    {
+        object GetAt(uint index);
+        bool IndexOf(object value, out uint index);
+        uint Size { get; }
+    }
+}
+
+namespace ABI.Microsoft.UI.Xaml.Interop
+{
+    [DynamicInterfaceCastableImplementation]
+    [Guid("036D2C08-DF29-41AF-8AA2-D774BE62BA6F")]
+    internal unsafe interface IBindableIterable : global::Microsoft.UI.Xaml.Interop.IBindableIterable, ABI.System.Collections.IEnumerable
+    {
+        
+    }
+
+    [DynamicInterfaceCastableImplementation]
+    [Guid("6A1D6C07-076D-49F2-8314-F52C9C9A8331")]
+    internal unsafe interface IBindableIterator : global::Microsoft.UI.Xaml.Interop.IBindableIterator
+    {
+        [Guid("6A1D6C07-076D-49F2-8314-F52C9C9A8331")]
+        public struct Vftbl
+        {
+            internal IInspectable.Vftbl IInspectableVftbl;
+            private void* _get_Current_0;
+            public delegate* stdcall<IntPtr, IntPtr*, int> get_Current_0 { get => (delegate* stdcall<IntPtr, IntPtr*, int>)_get_Current_0; set => _get_Current_0 = value; }
+            private void* _get_HasCurrent_1;
+            public delegate* stdcall<IntPtr, byte*, int> get_HasCurrent_1 { get => (delegate* stdcall<IntPtr, byte*, int>)_get_HasCurrent_1; set => _get_HasCurrent_1 = value; }
+            private void* _MoveNext_2;
+            public delegate* stdcall<IntPtr, byte*, int> MoveNext_2 { get => (delegate* stdcall<IntPtr, byte*, int>)_MoveNext_2; set => _MoveNext_2 = value; }
+
+            private static readonly Vftbl AbiToProjectionVftable;
+            public static readonly IntPtr AbiToProjectionVftablePtr;
+
+            static unsafe Vftbl()
+            {
+                AbiToProjectionVftable = new Vftbl
+                {
+                    IInspectableVftbl = global::WinRT.IInspectable.Vftbl.AbiToProjectionVftable,
+
+                    _get_Current_0 = (delegate*<IntPtr, IntPtr*, int>)&Do_Abi_get_Current_0,
+                    _get_HasCurrent_1 = (delegate*<IntPtr, byte*, int>)&Do_Abi_get_HasCurrent_1,
+                    _MoveNext_2 = (delegate*<IntPtr, byte*, int>)&Do_Abi_MoveNext_2
+
+                };
+                var nativeVftbl = (IntPtr*)ComWrappersSupport.AllocateVtableMemory(typeof(Vftbl), Marshal.SizeOf<global::WinRT.IInspectable.Vftbl>() + sizeof(IntPtr) * 3);
+                Marshal.StructureToPtr(AbiToProjectionVftable, (IntPtr)nativeVftbl, false);
+                AbiToProjectionVftablePtr = (IntPtr)nativeVftbl;
+            }
+
+
+            [UnmanagedCallersOnly]
+
```

Comment:
nit: blank line?

Reply:
Should I remove these? These appear as these files had conditional statements on NET5 and NETSTANDARD2.0 and I splitted the files through a script

Reply:
there is quite a bit of those, fine to leave as is.